src/main/java/me/senseiwells/essentialclient/clientscript/definitions/ItemStackDef.java

```diff
@@ -152,15 +152,18 @@ public List<MemberFunction> defineMethods() {
 			MemberFunction.of("isStackable", this::isStackable),
 			MemberFunction.of("getMaxCount", this::getMaxCount),
 			MemberFunction.of("asEntity", this::asEntity),
+			MemberFunction.of("getName", this::getName),
 			MemberFunction.of("getCustomName", this::getCustomName),
 			MemberFunction.of("isNbtEqual", 1, this::isNbtEqual),
 			MemberFunction.of("getNbt", this::getNbt),
 			MemberFunction.of("getNbtAsString", this::getNbtAsString),
 			MemberFunction.of("getTranslatedName", this::getTranslatedName),
 			MemberFunction.of("getMiningSpeedMultiplier", 1, this::getMiningSpeedMultiplier),
-
+			MemberFunction.of("clone", this::clone),
```

Comment:
You shouldn't implement a clone method like this. You should instead override the `copy` method in the class.